### PR TITLE
Gildas: fixed support for El Capitan.

### PR DESCRIPTION
**Pull request by bardeau**
This commit introduces fix for back-supporting El Capitan and probably older OSX versions. Can someone please accept this PR?

### CODE REVIEW

File: science/gildas/Portfile [hunk 1]
```diff
@@ -59,7 +62,7 @@ require_active_variants gtk2 x11 quartz
 compilers.enforce_some_fortran cfitsio
 
 compilers.choose    fc
-compilers.setup     -g95 -gcc44 -gcc45 -gcc46 require_fortran
+compilers.setup     -gcc44 -gcc45 -gcc46 require_fortran
```
Comment (ryandesign) [inline science/gildas/Portfile:65]: Was this change intentional? It doesn't look like it's related to the sort issue.

**Comment by ryandesign** [comment]
Thanks. The commit message should have this line (after the current commit message and a blank line):

```
Closes: https://trac.macports.org/ticket/60051
```

File: science/gildas/Portfile (hunk 1 above)
Comment (bardeau) [reply science/gildas/Portfile:65]: Yes it was intentional. Just cleaned this obsolete stuff by going through the Portfile. No, it is not directly related to the issue. Should I undo this part?

**Comment by bardeau** [comment]
> Thanks. The commit message should have this line (after the current commit message and a blank line):
> 
> ```
> Closes: https://trac.macports.org/ticket/60051
> ```

Actually it should fix both tickets https://trac.macports.org/ticket/60051 and https://trac.macports.org/ticket/60639 (can I put 2 "Closes" statements?), but I could not test directly under the relevant OSX versions. Did the automatic builds succeeded?

File: science/gildas/Portfile (hunk 1 above)
Comment (ryandesign) [reply science/gildas/Portfile:65]: Since it's unrelated, I'd put it in a separate commit so that you have an opportunity to describe the change separately. It's fine if the second commit is also in this PR.

**Comment by ryandesign** [comment]
> Actually it should fix both tickets https://trac.macports.org/ticket/60051 and https://trac.macports.org/ticket/60639 (can I put 2 "Closes" statements?)

Ok great. Yes you can; put one per line.

> but I could not test directly under the relevant OSX versions. Did the automatic builds succeeded?

They timed out after 50 minutes, and also they only test on recent macOS versions.

**Comment by bardeau** [comment]
Ok, I have removed the -g95 edit, I will do this properly later. I have put the ticket "Closes" (are they OK?).

**Comment by ryandesign** [comment]
Looks good! Successful builds on [10.9](https://build.macports.org/builders/ports-10.9_x86_64-builder/builds/121439), [10.10](https://build.macports.org/builders/ports-10.10_x86_64-builder/builds/117956) and [10.11](https://build.macports.org/builders/ports-10.11_x86_64-builder/builds/1163650). 10.7 and 10.8 will take awhile since they're busy building other things but I expect they will continue to fail due to https://trac.macports.org/ticket/59753.
